src/Uno.UI/UI/Xaml/Media/Animation/ColorKeyFrameCollection.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Windows.UI.Xaml.Media.Animation
+{
+	partial class ColorKeyFrameCollection : DependencyObjectCollection<ColorKeyFrame>, IList<ColorKeyFrame>, IEnumerable<ColorKeyFrame>
+	{
+		public ColorKeyFrameCollection() : base(null, false)
+		{
+		}
+
+		internal ColorKeyFrameCollection(DependencyObject owner, bool isAutoPropertyInheritanceEnabled) : base(owner,  isAutoPropertyInheritanceEnabled)
+		{
+		}
+
+		// Following members are for binary backward compatibility. They can be safely remove in an eventual "breaking" new version.
```

Comment:
Not sure I understand this comment. How is binary backward compatibility problematic for something that was never before implemented? Was the generated code wrong?